xcdat/regridder/accessor.py

```diff
@@ -76,22 +76,15 @@ def grid(self) -> xr.Dataset:
 
         return ds
 
-    def _get_axis_data(
-        self, name: str, standard_name: str
-    ) -> Tuple[xr.DataArray, xr.DataArray]:
-        try:
-            axis = self._ds.cf[name]
-        except KeyError:
-            raise KeyError(
-                f"{standard_name} axis could not be correctly identified in the Dataset"
-            )
```

Comment:
This logic is already implemented by `get_coord_var()` so I reused that function here.

The `standard_name` kwarg isn't necessary because `get_axis_coord()` tries to find the coord var using `axis`, `standard_name`, or dim name (e.g., "lat").  Refer to `axis.CF_NAME_MAP`.